notebook/04_portfolio_trade.ipynb

```diff
@@ -4,378 +4,337 @@
+    "# gQuant - Making Quantitative Analysis Faster\n",
     "\n",
-    "It is very time consuming for the CPU to crunch massive amount of data and compute the complicated data science models. \n",
-    "Large data set requires distributed computation, which is too complicated for data scientist to adopt. \n",
-    "The quantitative workflow becomes more complicated than before. It integrates massive data from different sources and requires a lot of iterations to find the best approach. \n",
-    "gQuant is developed to handle all these above challenges by organizing a group of dataframes in a graph. It introduces the idea of \"dataframe-flow\" that manipulate the data at the graph level. The dataframe manipulation is organized into an acyclic directed graph, where the nodes are dataframe processors and the edges are the directions of passing resulting dataframes. This graph approach organizes the quant's workflow at a high level that addresses the complicated workflow challenge. gQuant is built on top of the NVIDIA's RAPIDS library, which can passes GPU cuDF dataframes in the graph. In this way, all data are stored in the GPU memory and manipulated in the GPU. We can get orders of magnitude performance boosts compared to CPU. gQuant dataframe-flow is dataframe agnostic. Switching to dask_cudf dataframe, the computation  automatically becomes multiple nodes and multiple GPUs distributed computation. \n",
+    "## Background\n",
+    "By definition, **Quantitative Finance** is the use of mathematical models and large datasets to analyze financial markets and securities, requiring massive computation to extract insight from the data. \n",
     "\n",
-    "In this blog, we will use a simple toy example to show how easy it is to accelerate the quant workflow in the GPU.\n",
+    "Many data science toolkits have been developed to help data scientists to manipulate the data. It starts with scalar number computations at the beginning. Later, the development of **Numpy** library helps to operate the numbers at vectors, and the popular **Pandas** library operates at a dataframe level. Manipulating data at a high level brings productivity gain for data scientists in quantitative finance.\n",
```

Comment:
Add url links to the numpy and pandas library.

Reply:
Done.